### ISSUE TITLE
Random crashes in Thaumonomicon

### ISSUE THREAD

**Issue by JWTHDYTWA** [Azanor/thaumcraft-beta#1591]
[I occasionally receive game crash when i read Thaumonomicon (then only).
Minecraft: 1.12.2
TC: 6.1BETA26](https://gist.github.com/Warrock3280/c4377d3ff2875e0a48dace12f45d164e)

**Comment by Marquisinator** [comment]
This happens to me too, when I look at certain recipes.

**Comment by Marquisinator** [comment]
I looked at your logs, and we're both having the same problem with rendering the Thaumonomicon.

**Comment by 256leon** [comment]
tried disabling the other mods? maybe a conflict with another mod?

**Comment by JWTHDYTWA** [comment]
@256leon No. I'd rather disable Minecraft. I have 400+ mods in my modpack. However, it's not compatibility issue for sure. Else it would be other mod somewhere in main part of the crashlog.
![screenshot_2018-12-30 crash-2018-12-19_21 40 32-client txt](https://user-images.githubusercontent.com/30724185/50549132-db407c00-0c89-11e9-8358-288396c09197.png)

**Comment by JWTHDYTWA** [comment]
At most in theory it could be some addon for TC, but not random mod. There is Thaumic Periphery, Solar Expansion, Baubles and DynamicTrees Compat For TC.

**Comment by JWTHDYTWA** [comment]
And only mod that affects rendering in gui is Blur.
